examples/cloud_storage/versions.tf

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Let's drop this file, we no longer require `version.tf` for examples and we don't want to restrict TPG 5.